src/components/Entities/Entity.scss

```diff
@@ -5,69 +5,91 @@
-  &--heart::before      { content: "❤️"; }
-  &--houseA::before     { content: "🏠"; }
-  &--houseB::before     { content: "🏡"; }
-  &--invisible::before  { content: "  "; }
-  &--leavesA::before    { content: "🍃"; }
-  &--leavesB::before    { content: "🍂"; }
-  &--leavesC::before    { content: "🌿"; }
-  &--mart::before       { content: "🏪"; }
-  &--monkey::before     { content: "🐒"; }
-  &--musichall::before  { content: "🏫"; }
-  &--palm::before       { content: "🌴"; }
-  &--person::before     { content: "🏃"; }
-  &--personWalk::before { content: "🚶"; }
-  &--police::before     { content: "🚓"; }
-  &--rabbit::before     { content: "🐇"; }
-  &--santa::before      { content: "🎅"; }
-  &--shell::before      { content: "🐚"; }
-  &--shit::before       { content: "💩"; }
-  &--silverware::before { content: "🍴"; }
-  &--snake::before      { content: "🐍"; }
-  &--snowflake::before  { content: "❄"; }
-  &--snowman::before    { content: "⛄️"; }
-  &--speedboat::before  { content: "🚤"; }
-  &--moai::before       { content: "🗿"; }
-  &--sun::before        { content: "☀"; }
-  &--sunglasses::before { content: "👓"; }
-  &--tape::before       { content: "📼"; }
-  &--taxi::before       { content: "🚕"; }
-  &--tornado::before    { content: "💨"; }
-  &--treeA::before      { content: "🌲"; }
-  &--treeB::before      { content: "🎄"; }
-  &--treeC::before      { content: "🌳"; }
-  &--turtle::before     { content: "🐢"; }
-  &--wave::before       { content: "🌊"; }
-  &--willows::before    { content: "🌾"; }
+  .twemoji &::before, &::after {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-size: auto;
+  }
+
+  $entities: (
+    ambulance: "🚑",
+    banana: "🍌",
+    bee: "🐝",
+    bike: "🚴",
+    boots: "👢",
+    building: "🏢",
+    cactus: "🌵",
+    camel: "🐫",
+    carA: "🚗",
+    carB: "🚙",
+    chicken: "🐣",
+    chopper: "🚁",
+    cloud: "☁",
+    corn: "🌽",
+    creepsun: "🌞",
+    dancer: "💃",
+    door: "🚪",
+    elephant: "🐘",
+    fire: "🔥",
+    firetruck: "🚒",
+    fishA: "🐟",
+    fishB: "🐠",
+    flowerA: "🌻",
+    flowerB: "🌹",
+    flowerC: "🌷",
+    gator: "🐊",
+    ghost: "👻",
+    hammer: "🔨",
+    heart: "❤️",
+    houseA: "🏠",
+    houseB: "🏡",
+    invisible: "  ",
+    leavesA: "🍃",
+    leavesB: "🍂",
+    leavesC: "🌿",
+    mart: "🏪",
+    monkey: "🐒",
+    musichall: "🏫",
+    palm: "🌴",
+    person: "🏃",
+    personWalk: "🚶",
+    police: "🚓",
+    rabbit: "🐇",
+    santa: "🎅",
+    shell: "🐚",
+    shit: "💩",
+    silverware: "🍴",
+    snake: "🐍",
+    snowflake: "❄",
+    snowman: "⛄️",
+    speedboat: "🚤",
+    moai: "🗿",
+    sun: "☀",
+    sunglasses: "👓",
+    tape: "📼",
+    taxi: "🚕",
+    tornado: "💨",
+    treeA: "🌲",
+    treeB: "🎄",
+    treeC: "🌳",
+    turtle: "🐢",
+    wave: "🌊",
+    willows: "🌾"
+  );
+
+  @each $entity, $value in $entities {
+    &--#{$entity}::before { content: $value; }
+
+    @if $entity != "snowman" { // twemoji doesn't seem to have a snowman :(
```

Comment:
What happens if we don't have this :corn:ditional? Would the default unicode character not show up since `content: '';` here? I'd really prefer to drop this conditional if possible.